src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -151,3 +156,66 @@ def signature_hash_algorithm(self):
             raise UnsupportedAlgorithm(
                 "Signature algorithm OID:{0} not recognized".format(oid)
             )
+
+    @property
+    def extensions(self):
+        # TODO: remove this when we finish landing support for extensions.
+        warnings.warn(
+            "Extension support is not fully implemented. Unsupported "
+            "extensions are IGNORED at this time."
+        )
```

Comment:
:+1:

On Sun, Mar 22, 2015 at 11:11 AM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/hazmat/backends/openssl/x509.py
> <https://github.com/pyca/cryptography/pull/1769#discussion_r26903957>:
>
> > @@ -151,3 +156,66 @@ def signature_hash_algorithm(self):
> >              raise UnsupportedAlgorithm(
> >                  "Signature algorithm OID:{0} not recognized".format(oid)
> >              )
> > +
> > +    @property
> > +    def extensions(self):
> > +        # TODO: remove this when we finish landing support for extensions.
> > +        warnings.warn(
> > +            "Extension support is not fully implemented. Unsupported "
> > +            "extensions are IGNORED at this time."
> > +        )
>
> We do: #1743 <https://github.com/pyca/cryptography/issues/1743>. I need
> to mark which ones I'd consider required before we start raising exceptions
> but that's the complete list from the RFC.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1769/files#r26903957>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084